README.md

```diff
@@ -1,4 +1,5 @@
-![parse-repository-header-sdk-flutterdart](https://user-images.githubusercontent.com/5673677/166121476-3838ac9f-a437-409c-bb28-6e1dfcc36b1c.png)
+![parse-repository-header-sdk-dart](https://user-images.githubusercontent.com/5673677/166120960-ea1f58e3-a62b-4770-b541-f64186859339.png)
```

Comment:
What should this then be officially called? The Parse Dart SDK or the Parse Flutter SDK? Both names are currently used in this PR.

The repository itself is called parse-sdk-flutter.

Reply:
well, it should be Parse Dart SDK because all the code in the SDK uses pure Dart.

Reply:
Should be create a new repository Parse-SDK-Parse and archive this one?

Reply:
I do not know the consequence of renaming the repo. the one I know for sure is that all the people will have a broken upstream remote git-URL

will that create a problem when we try to publish to pub.dev?

Reply:
Renaming a repo on GitHub won't break the old URL. GitHub maintains a redirect.

Reply:
> I just don't know whether the registry has a concept for renaming where is also redirects.

I did not follow you here, can you tell me what a registry is in this context

Reply:
Thank you, by the way, it was my question too, I think it would be better to change the repository to a better name.

Reply:
I also agree with ```parse_sdk_dart```